src/DynamoCoreWpf/ViewModels/Core/WorkspaceViewModel.cs

```diff
@@ -936,14 +936,18 @@ internal void FitViewInternal()
                 maxY = GetSelectionMaxY();
             }
             else
-            {   // no selection, fitview all nodes
+            {   // no selection, fitview all nodes and notes
                 if (!_nodes.Any()) return;
 
                 List<NodeModel> nodes = _nodes.Select(x => x.NodeModel).Where(x => x != null).ToList();
```

Comment:
In which case `x.NodeModel` will be `null`? Could you try if it is fine to remove `.Where(x => x != null)`?

`NodeModel` and `NoteModel`'s base class is `ModelBase`, so we may concatenate two lists together. Following `minX`, `maxX`, `minY`, `maxY` need to iterate the list 4 times, replacing it with a simple for-loop so that we can do that in one iteration.